sh/upload.sh

```diff
@@ -47,40 +47,40 @@ _short_help() {
 
 ###################################################
 # Automatic updater, only update if script is installed system wide.
-# Globals: 4 variables, 2 functions
-#   Variables - INFO_FILE, COMMAND_NAME, LAST_UPDATE_TIME, AUTO_UPDATE_INTERVAL
-#   Functions - _update, _update_config
+# Globals: 5 variables, 2 functions
+#   COMMAND_NAME, REPO, INSTALL_PATH, TYPE, TYPE_VALUE | _update, _update_value
 # Arguments: None
 # Result: On
 #   Update if AUTO_UPDATE_INTERVAL + LAST_UPDATE_TIME less than printf "%(%s)T\\n" "-1"
 ###################################################
 _auto_update() {
     (
-        [ -w "${INFO_FILE}" ] && . "${INFO_FILE}" && command -v "${COMMAND_NAME}" 2>| /dev/null 1>&2 && {
-            [ "$((LAST_UPDATE_TIME + AUTO_UPDATE_INTERVAL))" -lt "$(date +'%s')" ] &&
-                _update 2>&1 1>| "${INFO_PATH}/update.log" &&
-                _update_config LAST_UPDATE_TIME "$(date +'%s')" "${INFO_FILE}"
-        }
+        _REPO="${REPO}"
```

Comment:
Codacy found an issue: [Possible misspelling: REPO may not be assigned, but _REPO is.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6201392)